src/goose.rs

```diff
@@ -1628,6 +1628,19 @@ impl GooseUser {
             self.send_request_metric_to_parent(request_metric.clone())?;
         }
 
+        if request.error_on_fail && !request_metric.success {
+            if let Ok(r) = response {
+                if let Ok(body) = r.text().await {
+                    if !body.is_empty() {
+                        error!("{:?} {}, {}", &path, &request_metric.error, &body);
```

Comment:
This information is already available if you enable `--debug-log`. I'd much prefer to keep it there, as this can lead to huge errors on the console that are difficult to review. This will simplify this block of code.